modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -36,18 +36,33 @@ static inline float tsdfToFloat(TsdfType num)
 {
     return float(num) * (-1.f / 128.f);
 }
-
+*/
 HashTSDFVolume::HashTSDFVolume(float _voxelSize, cv::Matx44f _pose, float _raycastStepFactor,
-                               float _truncDist, int _maxWeight, float _truncateThreshold,
-                               int _volumeUnitRes, bool _zFirstMemOrder)
+    float _truncDist, int _maxWeight, float _truncateThreshold,
+    int _volumeUnitRes, bool _zFirstMemOrder)
     : Volume(_voxelSize, _pose, _raycastStepFactor),
-      maxWeight(_maxWeight),
-      truncateThreshold(_truncateThreshold),
-      volumeUnitResolution(_volumeUnitRes),
-      volumeUnitSize(voxelSize * volumeUnitResolution),
-      zFirstMemOrder(_zFirstMemOrder)
+    maxWeight(_maxWeight),
+    truncateThreshold(_truncateThreshold),
+    volumeUnitResolution(_volumeUnitRes),
+    volumeUnitSize(voxelSize* volumeUnitResolution),
+    zFirstMemOrder(_zFirstMemOrder),
+    volDims(volumeUnitResolution, volumeUnitResolution, volumeUnitResolution)
```

Comment:
If `volDims` is always equal to `volumeUnitResolution`x3 then we don't need this member